locust/main.py

```diff
@@ -183,7 +179,8 @@ def is_valid_percentile(parameter):
         options.spawn_rate = options.hatch_rate
 
     if options.legacy_ui:
-        sys.stderr.write("[DEPRECATED] The legacy UI is deprecated and will be removed soon\n")
+        sys.stderr.write("[DEPRECATED] The legacy UI is deprecated")
```

Comment:
I think "deprecated" usually means "going to be deleted, but still works", so this line is confusing :)